x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -112,7 +112,7 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables Discover embedded within timeline
    *
    * */
-  discoverInTimeline: true,
+  discoverInTimeline: false,
```

Comment:
Can you do this in a separate PR and merge it directly to main and the 8.10 branch once it's created?

Reply:
Sure. 👍